conf/base.config

```diff
@@ -10,9 +10,9 @@
 
 process {
 
-    cpus   = { check_max( 1    * task.attempt, 'cpus'   ) }
-    memory = { check_max( 6.GB * task.attempt, 'memory' ) }
-    time   = { check_max( 4.h  * task.attempt, 'time'   ) }
+    cpus   = { 1      * task.attempt }
```

Comment:
Can someone point me to why this is not required anymore?

Reply:
Until nextflow 24.04, nextflow itself did not have a concept of upper bounds for resource requests. This is why nf-core came up with the `check_max` function. As of 24.04, this is implemented in nextflow and thus the `check_max` function is obsolete. Check the nextflow release notes [here](https://nextflow.io/blog/2024/nextflow-2404-highlights.html#:~:text=them%20in%20action!-,Resource%20limits,-The%20resourceLimits%20directive).